src/bundle.ts

```diff
@@ -52,7 +52,7 @@ const _bundle = (
       if (!_path || !inventoryPath || !inventoryRef) return;
 
       const bundled$Ref = get(document, _path);
-      if (bundled$Ref) {
+      if (bundled$Ref !== void 0) {
```

Comment:
Ideally, we performed a 'has' call first and then proceeded, but there is no way to represent `undefined` in both JSON & YAML, so we're safe.

https://github.com/stoplightio/json/blob/886aeaf21146bc14484b371dadd88c7ed69ffddc/src/bundle.ts#L71 - I considered adding `&& target !== null`, yet it's rather redundant as checks we have prior to that should already terminate the process.